### PR TITLE
Moved files into package and parameterised arguments.

### PR DESCRIPTION
**Pull request by aslanbekirov**
packaging added because of easy management for further development.

### CODE REVIEW

File: java/README.md~ [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Java Rendler Framework:
```
Comment (ssk2) [inline java/README.md~]: This file shouldn't have been checked in.

File: java/README.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ Java Rendler Framework:
 <br> 1) cd hostfiles/java </br> 
 <br> 2) mvn clean compile assembly:single </br>
 <br> 3) export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib </br>
-<br> 4) java -cp target/rendler-1.0-SNAPSHOT-jar-with-dependencies.jar RendlerMain 127.0.1.1:5050 250 </br>
+ <br> # Run RendlerMain with parameter 'master' or parameters 'master' 'tasks' 'url'. In former, default values for tasks and url are 5 and "http://mesosphere.com", respectively.</br>
```
Comment (ssk2) [inline java/README.md]: Whitespace at beginning of line.

**Comment by ssk2** [comment]
Can you squash these commits down? Also - it's not part of this PR but it'd be nice if we could use actual Markdown syntax in `java/README.md` - not sure why it uses HTML.

File: java/README.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ Java Rendler Framework:
...
+<br> 4) java -cp target/rendler-1.0-SNAPSHOT-jar-with-dependencies.jar RendlerMain 127.0.1.1:5050</br>
```
Comment (ssk2) [inline java/README.md]: This doesn't reference any of the parameters you mention in the line above. I would remove `In former, default values for tasks and url are 5 and "http://mesosphere.com", respectively.` and just put them in this example command itself.

File: java/README.md~ (hunk 1 above)
Comment (aslanbekirov) [inline java/README.md~]: Yes it was incidently, my fault.

File: java/README.md (hunk 3 above)
Comment (aslanbekirov) [inline java/README.md]: Thanks  for comments. I changed them as below
<br> # Run RendlerMain with parameter 'master' or parameters 'master' 'tasks' 'url'.</br>
<br> 4) java -cp target/rendler-1.0-SNAPSHOT-jar-with-dependencies.jar RendlerMain 127.0.1.1:5050 100 http://mesosphere.com</br>

File: java/README.md~ (hunk 1 above)
Comment (ssk2) [inline java/README.md~]: Can you remove it? If you then squash the commits, it won't show up.
See for reference: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

File: java/README.md [hunk 4]
```diff
@@ -3,6 +3,7 @@ Java Rendler Framework:
 <br> 1) cd hostfiles/java </br> 
 <br> 2) mvn clean compile assembly:single </br>
 <br> 3) export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib </br>
-<br> 4) java -cp target/rendler-1.0-SNAPSHOT-jar-with-dependencies.jar RendlerMain 127.0.1.1:5050 250 </br>
+<br> # Run RendlerMain with parameter 'master' or parameters 'master' 'tasks' 'url'.</br>
+<br> 4) java -cp target/rendler-1.0-SNAPSHOT-jar-with-dependencies.jar RendlerMain 127.0.1.1:5050 100 http://mesosphere.com</br>
```
Comment (ssk2) [inline java/README.md]: Let's change this file to use Markdown syntax instead of HTML.

**Comment by elingg** [comment]
Thanks @aslanbekirov! Could you please revert the formatting changes so we can clearly see which changes were made? If you would like to make formatting changes, could you create a separate PR. Typically we break formatting changes into a separate PR for faster review.

**Comment by aslanbekirov** [comment]
@elingg, the  Mesosphere-JavaFormatter.xml contains line 
\<setting id="org.eclipse.jdt.core.formatter.comment.line_length" value="80"/\>, that is why, while packaging with maven the format appears as I committed which differs from github version. Do I have to change this value or am I missing something?

**Comment by elingg** [comment]
@aslanbekirov, This does make sense. However, could you check in your logic changes (i.e. updating the readme/parameterizing the starting URL) as a separate PR from your packaging and formatting changes? This way I can more quickly review your code instead of going through each commit separately. I prefer to make formatting/packaging changes into a separate PR for more clarity and to avoid introducing bugs.

**Comment by elingg** [comment]
@aslanbekirov, is this safe to close since you have been splitting up this PR into multiple PR's?

**Comment by aslanbekirov** [comment]
@elingg yes, we can close. Thanks.

**Pull request closed without merging**